front/lib/api/wrappers.ts

```diff
@@ -134,7 +134,12 @@ export function withSessionAuthenticationForWorkspace<T>(
 
 /**
  * This function is a wrapper for Public API routes that require authentication for a workspace.
- * It must be used on all routes that require workspace authentication (prefix: /v1/w/[wId/]).
+ * It must be used on all routes that require workspace authentication (prefix: /v1]).
+ *
+ * opts.isWorkspaceRoute is used to determine if the route is under the /v1/w/[wId/] prefix.
```

Comment:
🤔 Personally not a big fan of this flag, either we implement a helper that does't not strictly enforces that a workspace exist in the URL or if it's only one endpoint you can directly do your own stuff there.